crates/biome_cli/src/execute/mod.rs

```diff
@@ -60,7 +60,7 @@ impl Execution {
 }
 
 impl Execution {
-    pub(crate) fn to_features(&self) -> Vec<FeatureName> {
+    pub(crate) fn to_features(&self) -> FeatureName {
```

Comment:
Should we rename this?

```suggestion
    pub(crate) fn to_feature(&self) -> FeatureName {
```